### ISSUE TITLE
Log files are not deleted after file count reaches 1000

### ISSUE THREAD

**Issue by anarut** [bootique/bootique-logback#54]
**sample configuration**:

```
log:
  level: warn
  loggers:
    io.bootique.logback:
      level: info
  appenders:
    - type: file
      logFormat: '[%d{"dd/MMM/yyyy:HH:mm:ss,SSS"}] %t %X{txid} %X{principal:-?} %-5p %c{1}: %m%n%ex'
      file: 'logs/server.log'
      rollingPolicy:
        type: sizeAndTime
        fileNamePattern: 'logs/server.%d{yyyy-MM-dd}.%i.log'
        historySize: 10
        fileSize: 2MB
        totalSize: 5MB
```

**to reproduce**:  send logs in infinite loop

After creating server.2020-10-09.**1000**.log file automatic delete is stopped.

**Comment by stariy95** [comment]
This one is a bug in Logback itself, it's fixed in a 1.3.0-alpha once it's GA we could update. See https://jira.qos.ch/browse/LOGBACK-1297

**Comment by stariy95** [comment]
Fix is back-ported to the 1.2.4 version (https://jira.qos.ch/browse/LOGBACK-1175), so it should be all good now.

**Comment by andrus** [comment]
Should probably add a note to release notes
